src/components/Authorization/Authorization.jsx

```diff
@@ -32,114 +33,168 @@ export const Authorization = ({primary}) => {
     event.preventDefault();
     setEmailError("");
     setPasswordError("");
-
+    let isValid = true;
     if (!email) {
       setEmailError("Это поле обязательно для заполнения");
```

Comment:
DRY - don't repeat yourself. You should place this text to constant and reuse it. Better if we try to create object with properties like this:
```
const validationErrors = {
  required: "Это поле обязательно для заполнения",
  emailFormat: "Вы заполняете поле в неверном формате",
  passwordLength: "Пароль должен содержать минимум 8 символов",
}
```

Check this in another places.

Reply:
fix